### DISCUSSION TITLE
Help with NMDS plot customization

### DISCUSSION THREAD

**Discussion by jahid10** [vegandevs/vegan#573] [General]
Hi, Im relatively new in R and Vegan package.  I have performed NMDS on my taxonomic abundance data which gave me a 2 dimensional scatter plots. However, I want to have an oval shade on points where they are clusturing, like the image below. How Can I modify the code below to make my graph have this representation (given that I already plotted up to the scatter graph
![Screenshot_7](https://user-images.githubusercontent.com/65849689/233362314-95ed618f-bc1d-4a0b-b3c9-7b4ce16940d1.jpg)

**Comment by jarioksa** [comment]
Check **vegan** function `ordiellipse(..., kind = "ehull")` that draws enclosing ellipses.
